src/lib/i18next/translator.ts

```diff
@@ -0,0 +1,27 @@
+import { useTranslation } from 'react-i18next';
+
+/* Encoder between Zawgyi and Unicode */ 
+import { ZawgyiConverter } from './encodings'; 
+const converter = new ZawgyiConverter(); 
+
+enum LanguageEncoding { 
+    UNICODE,
+    ZAWGYI 
+}
+
+/* Global variable that determines user encoding */
+// TODO Step 4: Interpret encoding based on user settings
+let userEncoding: LanguageEncoding = LanguageEncoding.UNICODE; 
+
+export const useIntl = () => {
```

Comment:
NIT (non-blocker): I was trying to think of a better hook name because I think `useIntl` isn't very expressive for devs trying to pick this up (since they likely won't know what intl stands for). I couldn't think of a good alternative, but if you can think of anything more descriptive before you merge, that'd be great.

Reply:
Gotcha. I'll change it to `useInternationalization` instead, as that seems more clear while keeping things concise. If we're _really_ going for clarity, I can change it to `useLanguageInternationalization`, but that's a pretty long name for a hook.